README.md

```diff
@@ -135,6 +135,52 @@ The cluster creation and retrieving connection info can be done directly in comm
   hzc --cluster-name <CLUSTER NAME> --cloud-token <DISCOVERY TOKEN>
   ```
 
+## SSL Configuration
+
+You can use the following configuration file to enable SSL support:
+```
+ssl:
+    servername: "HOSTNAME-FOR-SERVER"
+    # or: insecureskipverify: true
```

Comment:
maybe we can delete `or:` part from this comment. also do we have a place where these config options are documented. we can point to that documentation in CLC docs.

Reply:
The idea was making the user aware of `insecureskipverify` option, which can be used when `servername` cannot be used. README is the only documentation for CLC. We should improve that.